docs/site/Server.md

```diff
@@ -58,6 +58,52 @@ export class HelloWorldApp extends RestApplication {
 
 ## Configuration
 
+### Enable HTTPS
+
+Enabling HTTPS for the LoopBack REST server is just a matter of specifying the
+protocol as `https` and specifying the crendential files. You can either use a
+key-certificate or pfx-passphrase combo.
+
+In the following app we confiure HTTPS for a bare miminum app using a
+key-certificate combo.
+
+```ts
+import {RestApplication, RestServer, RestBindings} from '@loopback/rest';
+import * as fs from 'fs';
+
+export async function main() {
+  const options = {
+    rest: {
+      protocol: 'https',
+      key: fs.readFileSync('./key.pem'),
+      cert: fs.readFileSync('./cert.pem'),
+    },
+  };
+  const app = new RestApplication(options);
+  app.handler(handler => {
+    handler.response.send('Hello');
+  });
+  await app.start();
+  const server = await app.getServer(RestServer);
+  const port = await server.get(RestBindings.PORT);
+  const protocol = await server.get(RestBindings.PROTOCOL);
+  console.log(`Server is running at ${protocol}://127.0.0.1:${port}`);
+}
+```
+
+The same app can be HTTPS configured using a pfx-passphrase combo by setting the
```

Comment:
This second example seems redundant for me, no need to duplicate Node.js core documentation.

If we decide to keep it:

 - Reword "pfx-passphrase combo", e.g. _The same application can be configured for HTTPS using a PFX file by setting the `rest` options to the following:_
 - Fix the code snippet to actually show `pfx` instead of `key` + `cert` 😉